### ISSUE TITLE
Probability and expected value

### ISSUE THREAD

**Issue by ammar1510** [cpinitiative/usaco-guide#3939]
Is it possible to add probability and expected value as a topic?

**Comment by bqi343** [comment]
There are some relevant resources here: https://usaco.guide/gold/combo#resources

**Comment by ammar1510** [comment]
Yes, but could you add it as a topic to USACO guide. There are problems on expected value beyond the gold level. 
It's a bit hard to find in depth resources to probability.

**Comment by SansPapyrus683** [comment]
i mean is it really important enough to warrant its own module?

**Comment by ammar1510** [comment]
well, there are topics such as bitsets and hashmaps. I think probability lies somewhere alongside those in terms of how frequent it is. Regardless, just my thought that probability is a nice excercise for improvement in competitive programming. Ultimately it's upto you to decide on it's importance.

**Comment by envyaims** [comment]
> i mean is it really important enough to warrant its own module?

i think EV should definitely be considered

**Comment by SansPapyrus683** [comment]
i've seen like one problem in gold that uses that
not sure about plat

**Comment by envyaims** [comment]
> i've seen like one problem in gold that uses that not sure about plat

its a common topic in CP in general

**Comment by SansPapyrus683** [comment]
well sure i'll leave this issue open, make a module if you want

**Comment by bqi343** [comment]
it's currently covered here: https://usaco.guide/gold/combo#expected-value

we can consider separating it into a new module if there are sufficiently many problems (currently there are only a few)
